server/events/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
```

Comment:
Don't forget to include the applications in the admin console, so that we can add/delete/modify events from the `django-admin` page.

Reply:
fixed